### ISSUE TITLE
Add ability to manually execute a health check

### ISSUE THREAD

**Issue by nvllsvm** [docker/cli#1479]
Docker defaults to 30 seconds before the first health check is run, but this value has proven to be overkill in my use. Containers often are often ready to check far sooner than that. The [interval](https://docs.docker.com/engine/reference/builder/#healthcheck) can be overridden to start the check sooner, but that would change the interval for every check - resulting in unnecessary load there after.

Being able to manually force the execution of a health check would solve this problem.

For context, I'm trying to ensure dependency containers are healthy before starting dependents. I've attempted to parse and execute the output of `docker inspect` but that gets fairly complex - especially for checks of type `CMD` with multiple arguments.

**Comment by thaJeztah** [comment]
Sounds like the discussion on this pull-request may be of interest to you; https://github.com/moby/moby/pull/33918. Unfortunately, it looks like the contributor does not currently have time to work on that PR, but perhaps someone in the community is interested in working on the continuation

**Comment by bsousaa** [comment]
Closing in favour of https://github.com/moby/moby/issues/33410.
